src/com/ableton/Pyenv.groovy

```diff
@@ -69,10 +73,11 @@ class Pyenv implements Serializable {
         pip install virtualenv
         virtualenv ${venv.venvRootDir}
       """,
-    )
-
-    if (result != 0) {
-      script.error "Installation of Python ${pythonVersion} failed with code ${result}"
+      )
+      if (result != 0) {
+        throw new Exception("Installation of Python ${pythonVersion} failed with code " +
+          "${result}")
+      }
```

Comment:
Actually, let's get rid of this entirely. We can drop `returnStatus: true` in the `sh` command and just have it be alone in the `retry` loop with nothing else.

Reply:
I like the idea. See 64d44acb31ad027322529a298d600f47473b4e41 which also removes the second sentence from the commit message.